### PR TITLE
#36 add unit test to assert no cwe.mitre.org links in the references section

### PR DESCRIPTION
**Pull request by robocoder**

### CODE REVIEW

File: tests/test_references.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import re
+
+from vulndb import DBVuln
+from tests.vulndb_test import VulnDBTest
+
+class TestReferences(VulnDBTest):
+
+    def test_no_redundant_cve_mitre_org_urls(self):
+        invalid = []
```
Comment (andresriancho) [inline tests/test_references.py:16]: To make this 100% complete please do add a doc to the test method that explains why this is bad, and what the developer should do to fix it.

**Comment by andresriancho** [comment]
Add doc and I'll merge

**Comment by andresriancho** [comment]
When this PR is closed I should also close https://github.com/vulndb/data/issues/36

**Comment by robocoder** [comment]
Docstring added.

**Pull request merged**